utkdigitalinitiatives/utk-fedora-fedoragsearch-solr-config
# Trac 875

This is a PULL request in support of JIRA TRAC-875.

New code from utk-fedora-fedoragsearch-solr-config(TRAC-875) should be merged to
utk-fedora-fedoragsearch-solr-config(institutional_repository)

Code changes are in /utk-fedora-fedoragsearch-solr-config/fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

Testing instructions:

    vagrant destroy --force
    git pull 
    git checkout -b remotes/origin/TRAC-875
    vagrant up
    -- TRACE(TRAC-875) will load the utk-fedora-fedoragsearch-solr-config(TRAC-875)

    userA: ingest one thesis. 
    thesis_manager: accept-publish
    admin: Check the url of the thesis to see the pid
    For example: http://localhost:8000/islandora/object/utk.ir.td%3A16
    If you are logged in as admin you can also go the the datastreams page.
    This item has PID  utk.ir.td:16

    Update the solr index so that you can find utk.ir.td:16
       http://localhost:8080/fedoragsearch/rest?operation=updateIndex

    List all pids in solr index on localhost
    http://localhost:8080/solr/collection1/select?q=*%3A*&sort=PID+asc&rows=1000000&fl=PID&wt=csv&indent=true

    Look at the data display at:
    http://localhost:8080/solr/collection1/select?q=PID%3A%22utk.ir.td%3A16%22&fl=utk_mods_*&wt=xml&indent=true

    Look at the at the solr schema browser at: http://localhost:8080/solr/#/collection1/schema-browser
    See if you can locate the fieldname: utk_mods_etd_author_s

@markpbaggett @CanOfBees
